pkg/controller/master/upgrade/upgrade_controller.go

```diff
@@ -49,7 +49,7 @@ const (
 	upgradeServiceAccount          = "system-upgrade-controller"
 	harvesterSystemNamespace       = util.HarvesterSystemNamespaceName
 	harvesterUpgradeLabel          = "harvesterhci.io/upgrade"
-	harvesterManagedLabel          = "harvesterhci.io/managed"
+	harvesterManagedLabel          = util.HarvesterManagedNodeLabelKey
```

Comment:
Same as above. Do we have to redefine these constants? Can't we just use the `util.*` constants?

Reply:
I did not intend to change too many existing codes; in the past, consts are defined here and there, instead of a public file.